at-luantran/Week3/exercise3.rb

```diff
@@ -0,0 +1,24 @@
+# Ruby data structure to hold this data.
+arr = [
+  {:name => "Paul Barry", :interest => ["Programming", "Networking", "Security", "Open Source", "Frameworks"]},
+  {:name => "Chris Meudec", :interest => ["Testing", "Safety Systems", "Formal Systems", "Programming Languages"]},
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>